pkg/awsutils/awsutils.go

```diff
@@ -562,8 +630,11 @@ func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
 		Tags:            tags,
 	}
 
+	start := time.Now()
 	_, err := tagSvc.TagResources(tagInput)
+	awsAPILatency.WithLabelValues("TagResources", fmt.Sprint(err != nil)).Observe(msSince(start))
 	if err != nil {
+		awsAPIErrInc("TagResources", err)
 		log.Warnf("Fail to tag the newly created eni %s  %v", eniID, err)
```

Comment:
If we're counting this as an Error, why do we log as a Warning?   Might be a good idea to make a function `logAndTallyErrorf` since we appear to always want them both together.